src/transforms/concat.js

```diff
@@ -122,7 +125,7 @@ module.exports = (code, exports, moduleMap) => {
         // let id = Number(match[1]);
 
         if (!path.scope.hasBinding(name) && !addedExports.has(name)) {
-          let {bindings} = path.scope;
+          let exports = moduleMap.get(+match[1]).cacheData.exports;
```

Comment:
Previously, all variables in the current scope would be placed on the exports namespace object, not just the actual exports of the module. Changed to get the exports from the module.